### DISCUSSION TITLE
TA Error on Launch; Does not start

### DISCUSSION THREAD

**Discussion by MrLazerface** [josephdadams/TallyArbiter#496] [Q&A]
Hi, I have tally arbiter installed on a raspberry Pi 4 (using the bitfocus Companion Pi image as the base), but it is throwing an error every time it starts. It worked fine after installing it initially, and after several subsequent reboots it was fine. I moved my whole setup to a different city (same router in use with a DHCP reservation for the Pi) for a show, and now I get this error when trying to run TA standalone or have it autostart via pm2:
`/usr/lib/node_modules/tallyarbiter/dist/sources/TSL.js:1
TypeError: packet_1.default.createParser is not a function
    at new TSL3TCPSource (/usr/lib/node_modules/tallyarbiter/dist/sources/TSL.js:80:39)
    at initializeSource (/usr/lib/node_modules/tallyarbiter/dist/index.js:1181:24)
    at loadConfig (/usr/lib/node_modules/tallyarbiter/dist/index.js:1168:13)
    at startUp (/usr/lib/node_modules/tallyarbiter/dist/index.js:185:5)
    at Object.<anonymous> (/usr/lib/node_modules/tallyarbiter/dist/index.js:2340:1)
    at Generator.next (<anonymous>)
`
There's also a giant wall of code that appears when launching TA, I have included it here if its helpful.
[Tally Arbiter Error.txt](https://github.com/josephdadams/TallyArbiter/files/9823639/Tally.Arbiter.Error.txt)

Any idea what is happening? I'm a CLI noob so detailed instructions would be very helpful/appreciated.

**Comment by MatteoGheza** [comment]
Thanks for providing error logs.
How did you install TallyArbiter? What version are you running?

**Comment by josephdadams** [comment]
@MatteoGheza if we downgrade the `packet` library to 0.0.6 that error should go away. It’s related to TSL 3.1 over TCP

**Reply by MatteoGheza** [reply]
Downgraded packet to 0.0.6

**Reply by MatteoGheza** [reply]
I think we can release a new "beta" in a few days, there are lots of fixes and improvements.
I'll open a new PR right now for a fix, then I'll work on fixing the cloud connection
